deployment/configMigration.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
```

Comment:
Don't put the tool in our module. Please put it in another place.

Reply:
Agree. It's a standalone script, we can move it else place.
Where's the best place in you opinion?

Reply:
```src/tool``` ?

Reply:
maybe in deployment/tool?
It's more about deployment, other than any component.